core/src/tests/parachain.rs

```diff
@@ -179,6 +180,31 @@ fn default_relay_builder(
 				.unwrap(),
 		)
 		.unwrap();
+	state
+		.insert_storage(
+			pallet_session::GenesisConfig::<RRuntime> {
```

Comment:
Providing this genesis config will eventually trigger the `on_new_session` logic for all the relevant pallets such as `session_info` which requires `Some` in `AccountKeys`.